### PR TITLE
Implement newer_than filter for aws nuke command

### PR DESCRIPTION
**Pull request by hongil0316**
## Description

Fixes https://github.com/gruntwork-io/cloud-nuke/issues/485. 

Testing with the newer-than filter (retrieve elements "newer" than 1 hour): 
image

Testing with the newer-than filter (retrieve elements "newer" than 1s):
image

TODO: we may want to reword these filters to `excludeAfter` and `includeAfter` to be consistent with the naming within the code. Also, it could be quite confusing to understand "older-than" and "newer-than". 

## TODOs

Read the [Gruntwork contribution guidelines](https://gruntwork.notion.site/Gruntwork-Coding-Methodology-02fdcd6e4b004e818553684760bf691e).

- [ ] Update the docs.
- [ ] Run the relevant tests successfully, including pre-commit checks.
- [ ] Ensure any 3rd party code adheres with our [license policy](https://www.notion.so/gruntwork/Gruntwork-licenses-and-open-source-usage-policy-f7dece1f780341c7b69c1763f22b1378) or delete this line if its not applicable.
- [ ] Include release notes. If this PR is backward incompatible, include a migration guide.
- [ ] Attention Grunts - if this PR adds support for a new resource, ensure the `nuke_sandbox` and `nuke_phxdevops` jobs in `.circleci/config.yml` have been updated with appropriate exclusions (either directly in the job or via the `.circleci/nuke_config.yml` file) to prevent nuking IAM roles, groups, resources, etc that are important for the test accounts.

## Release Notes (draft)

Added / Removed / Updated [X].

### CODE REVIEW

File: commands/cli.go [hunk 1]
```diff
@@ -152,6 +157,11 @@ func CreateCli(version string) *cli.App {
 					Usage: "Only inspect resources older than this specified value. Can be any valid Go duration, such as 10m or 8h.",
 					Value: "0s",
 				},
+				&cli.StringFlag{
```
Comment (MoonMoon1919) [inline commands/cli.go:160]: This looks like a duplicate - should it be `older-than`?

File: aws/types.go [hunk 2]
```diff
@@ -91,18 +91,20 @@ type Query struct {
...
 	ExcludeResourceTypes []string
-	ExcludeAfter         time.Time
+	ExcludeAfter         *time.Time
+	IncludeAfter         *time.Time
 	ListUnaliasedKMSKeys bool
 }
 
 // NewQuery configures and returns a Query struct that can be passed into the InspectResources method
-func NewQuery(regions, excludeRegions, resourceTypes, excludeResourceTypes []string, excludeAfter time.Time, listUnaliasedKMSKeys bool) (*Query, error) {
+func NewQuery(regions, excludeRegions, resourceTypes, excludeResourceTypes []string, excludeAfter, includeAfter *time.Time, listUnaliasedKMSKeys bool) (*Query, error) {
```
Comment (MoonMoon1919) [inline aws/types.go:100]: Will we need to update tests that use `NewQuery()` to include the new arg?

**Review by MoonMoon1919 — COMMENTED** [review]
Added a few comments

File: aws/types.go (hunk 2 above)
Comment (hongil0316) [reply aws/types.go:100]: Ah yes. Updated it.

File: commands/cli.go (hunk 1 above)
Comment (hongil0316) [reply commands/cli.go:160]: One is for the `aws` command and another one for `inspect-aws` comamnd.

**Review by MoonMoon1919 — APPROVED** [review]

**Pull request merged**